Add EIP: Bitcoin Proof-of-Stake

etan-status opened the pull request.

wjmelements commented:
Shouldn't this be a BIP?

emmanuel-awosika commented:
> Shouldn't this be a BIP?

Since the proposal requires buy-in from the Ethereum community (mainly node operators), it makes sense to put out an EIP first. The current spec does include a phase where a BIP is created to enable the transition, though. Relevant quote from the `Stage 3: Signalling in Bitcoin blocks` section: 

> When >= 30% of produced Ethereum blocks over a sliding 4-week period (`6*24*7*28` = 28'224 blocks) indicate support for this EIP by including a valid Bitcoin block hash in their graffiti, a Bitcoin Improvement Proposal (BIP) SHALL be created defining a version bit that represents support by miners to eventually transition to Ethereum Proof-of-Stake.

@wjmelements

abcoathup commented on EIPS/eip-####.md:
```diff
@@ -0,0 +1,99 @@
+---
```
```suggestion
---
eip: 7671
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

abcoathup commented on EIPS/eip-####.md:
```diff
@@ -0,0 +1,99 @@
+title: Bitcoin Proof-of-Stake
+description: Bitcoin transition from Proof-of-Work to Proof-of-Stake
+author: Etan Kissling (@etan-status)
+discussions-to: https://ethereum-magicians.org/t/eip-bitcoin-proof-of-stake/19477
```
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7671-bitcoin-proof-of-stake/19477
```

etan-status commented:
- Renamed document to reflect assigned EIP number
- Added Rationale, Backwards Compatibility and Security Considerations sections
- Fixed typos and incorrect computations for number of blocks, adjusted period from from 12-weeks to 13-weeks (quarter year)

SamWilsn commented:
\*checks date\* Let me know if you intend to pursue this.
